users/provision_role.tf

```diff
@@ -3,7 +3,7 @@
 # AWS resources in the users account.
 # ------------------------------------------------------------------------------
 
-data "aws_iam_policy_document" "iam_admin_role_assume_role_doc" {
+data "aws_iam_policy_document" "provision_account_role_assume_role_doc" {
```

Comment:
`provisionaccount_role_assume_role_doc`.  See also line 21.

Reply:
I did the same thing in the "shared-services" directory in #3!  I must remember incorrectly what we agreed to.